rest-api/model/participant_summary.py

```diff
@@ -27,12 +27,12 @@ class ParticipantSummary(Base):
   participantId = Column('participant_id', Integer, ForeignKey('participant.participant_id'),
                          primary_key=True, autoincrement=False)
   biobankId = Column('biobank_id', Integer, nullable=False)
-  firstName = Column('first_name', String(80), nullable=False)
-  middleName = Column('middle_name', String(80))
-  lastName = Column('last_name', String(80), nullable=False)
+  firstName = Column('first_name', String(255), nullable=False)
```

Comment:
Is this just a safeguard (seems sane), or are we matching a specific value in HealthPro or something (in which case could you comment)?

Reply:
PTC currently doesn't impose *any* limits on most of its string fields (with exceptions for things like zip code); as a result, some data entered by users (albeit with garbage values) is failing to sync to the RDR, since it won't fit into our participant summary. 

We agreed not having any limits was not a good idea; they asked if we could increase our field sizes to 255, and I said that was fine. They probably should choose to limit some of these fields further (having a > 80 character name doesn't really make sense), but if they *do* send us a value that exceeds 80 chars, they want us to accept it. Added a comment.